tests/functions/test_origin_response.py

```diff
@@ -11,54 +11,66 @@
 
 
 @pytest.mark.parametrize(
-    "test_input",
+    "original_uri, want_digest",
     [
-        "/some/repo/listing",
-        "/some/repo/repodata/repomd.xml",
-        "/some/repo/ostree/repo/refs/heads/ok/test",
+        ("/some/repo/listing", True),
+        ("/some/repo/repodata/repomd.xml", True),
+        ("/some/repo/ostree/repo/refs/heads/ok/test", False),
     ],
 )
-def test_origin_response_valid_headers(
-    test_input,
-    expected=[{"key": "Cache-Control", "value": "max-age={}".format(max_age)}],
-):
+def test_origin_response_valid_headers(original_uri, want_digest):
     event = {
         "Records": [
             {
                 "cf": {
                     "request": {
+                        "uri": "/some/repo/a4ef27d",
```

Comment:
Could the testdata please be more realistic?

1. In reality, there won't be a /some/repo prefix.  Could it please be removed, as this is misleading readers as to how the setup works.

2. Could we please use a full checksum rather than abbreviating?  I am worried abbreviating in the testdata is setting ourselves up for extra work later, e.g. if we introduce a feature which more strongly relies on the property that the keys are valid sha256 sums.

Reply:
Oops! Yeah, not sure what I was thinking with the path prefix. I'll replace it with a real sha256.